Examples/ResourceLeak/README.md

```diff
@@ -0,0 +1,28 @@
+## Procedure-local Bugs of Resource Leaks
+
```

Comment:
can you shorten the description and use code examples to reflect the 4 scenarios?

Reply:
Resolved in update